pfcpiface/p4rt_translator.go

```diff
@@ -404,6 +502,61 @@ func (t *P4rtTranslator) ParseAccessIPFromReadInterfaceTableResponse(resp *p4.Re
 	return nil, ErrOperationFailed("parse Access IP from P4Runtime response")
 }
 
+func (t *P4rtTranslator) BuildApplicationsTableEntry(pdr pdr, internalAppID uint8) (*p4.TableEntry, error) {
+	applicationsBuilderLog := log.WithFields(log.Fields{
+		"pdr": pdr,
+	})
+	applicationsBuilderLog.Trace("Building P4rt table entry for applications table")
+
+	tableID := t.tableID(TableApplications)
+	entry := &p4.TableEntry{
+		TableId: tableID,
+		// priority for UP4 cannot be greater than 65535
+		Priority: int32(math.MaxUint8 - pdr.precedence),
+	}
+
+	// srcIP/srcPort is always set as an application endpoint
```

Comment:
Looking at the `parsePDI` this seems to be false. `srcIP` is assigned to src or dst based on the source interface. Please, can you verify this?

Reply:
Will do, but I believe I already checked that

Reply:
You were right, Daniele. It's already fixed.